src/Simulation/Capacitance.jl

```diff
@@ -1,146 +1,125 @@
-function _get_abs_bias_voltage(det::SolidStateDetector{T}) where {T <: SSDFloat}
-    potentials::Vector{T} = map(c -> c.potential, det.contacts)
-    return (maximum(potentials) - minimum(potentials)) * u"V"
-end
-
+export calculate_mutual_capacitance
```

Comment:
Should this go into `SolidStateDetectors.jl` where we gather all other `export` statements?